engine/runtime/src/main/scala/org/enso/compiler/codegen/IrToTruffle.scala

```diff
@@ -1504,7 +1504,7 @@ class IrToTruffle(
               s"argument<${name.map(_.name).getOrElse(String.valueOf(position))}>"
 
             val section = value.location
-              .map(loc => source.createSection(loc.start, loc.end))
+              .map(loc => source.createSection(loc.start, loc.length))
```

Comment:
Ideally there should be a unit test that verifies this change.

Reply:
Well, just running our tests with assertions enabled has found this, isn't it enough if we enable them then?